ferrocene/doc/qualification-plan/src/development.rst

```diff
@@ -208,3 +208,42 @@ current status of a branch after the change.
 The GitHub issue is used to track the analysis performed by either the original
 author of the patch or the Release Manager, post mortems and conclusions, and
 any additional tests introduced to prevent future recurrences of the problem.
+
+
+External contributions
+----------------------
+
+CircleCI does not run PRs from forks by default, which is good because it
```

Comment:
This section doesn't fit the overall tone/writing style for the QP. I think you are right that this should go into internal procedures.

Perhaps it can be added to the `working-with-the-ci.rst`?

Reply:
Should we adapt the tone or should we put it in a different place?

Reply:
I moved it to a new page in the internal procedures.

Reply:
And reworded it.